src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/Compiler/AddSessionDomainConstraintPassTest.php

```diff
@@ -148,6 +152,9 @@ private function createContainer($sessionStorageOptions)
         $pass = new AddSessionDomainConstraintPass();
         $pass->process($container);
 
+        $container->setDefinition('.service_subscriber.fallback_container', new Definition(Container::class));
```

Comment:
That test class does not compile the container so the `RegisterServiceSubscribersPass` pass is not executed. Consequently, services that have `Psr\Container\ContainerInterface` as an argument that would normally be replaced by a smaller locator tag end up relying on the autowired global container, and on the deprecated alias, so they trigger a deprecation.